modules/standard/Errors.chpl

```diff
@@ -158,6 +158,13 @@ module Errors {
     }
   }
 
+  class ArrayOOMError: Error {
```

Comment:
A couple of things:
1. Probably best to mark this as unstable for the time being.
2. I think our naming conventions would point towards using ``ArrayOomError`` — not that I like the look if that myself.

As a possible alternative, would a more general ``OutOfMemoryError`` make sense? Or do we really just want an array-specific one?

Reply:
Alright, I've marked it unstable and renamed to `ArrayOomError`, where we talked offline with being fine with the name since it is being marked unstable.